Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -111,28 +111,26 @@ void App::Connect(int connectionId, Platform::Object^ target) {
 }
 
 void App::OnLaunched(LaunchActivatedEventArgs^ args) {
-    if (EbrApplicationLaunched(args) == true) {
-        _ApplicationMainLaunch(ActivationTypeNone, nullptr);
-    }
+    TraceVerbose(TAG, L"OnLaunched invoked");
```

Comment:
>TraceVerbose(TAG, L"OnLaunched invoked"); [](start = 4, length = 41)

These logs should be in their Ebr counterparts so we get these logs even with App.Xaml